RELEASES.md

```diff
@@ -40,6 +40,7 @@ Breaking changes
 - Stdlib `string.Stringingable` -> `string.Stringable` by @vkleen in https://github.com/tweag/nickel/pull/1180
 - Fix the type of `array.elem` by @yannham in https://github.com/tweag/nickel/pull/1223
 - Change the enum tag start delimiter from backtick to single-quote by @vkleen in https://github.com/tweag/nickel/pull/1279
+- `import` is now a statement, `import "foo.ncl" arg1 arg2` requires parenthesis now: `(import "foo.ncl") arg1 arg2`, see https://github.com/tweag/nickel/pull/1293
```

Comment:
It's a detail, but we should refrain from using statement I think. Everything is an expression in Nickel, including imports, which do have a value (as opposed to an if statement in e.g. C). It's not too bad in RELEASES.md, but just as a general note.

Reply:
I agree. We should probably rephrase it in RELEASES as well, although I'm not sure how to properly put it to words.